### ISSUE TITLE
Extraction of z- (Color) data from heat map image

### ISSUE THREAD

**Issue by Seil85** [ankitrohatgi/WebPlotDigitizer#154]
It would be quite nice if we can extract the z-values of a "heat map" image. The goal would be to get a x,y,z - data set (3D), which maps coordinates x,y to calibrated colour values of an image. As an example I attached a picture
![grafik](https://user-images.githubusercontent.com/35099704/46455493-9781c780-c7ab-11e8-9de5-b7cf542f9d4f.png)
  - Hereby the z-axis have to be calibrated in a separate step to map the colours to individual values. The x-y calibration is already working quite well. 
  - 2D-Sampling in x- and y-direction would ensure a regular sampling grid
  - the clue is to map / project the colours given in the colour bar in a correct and keen way.

**Comment by ankitrohatgi** [comment]
Yes, this comes up quite often (see #41 and #82), but unfortunately I have been too short on time to work on this :(

**Comment by adammaj1** [comment]
Is it done or stiil waits for the implementation ?
